js/src/weekly_report.js

```diff
@@ -0,0 +1,60 @@
+// # Generate a weekly report of completed issues per repo
+const fetch = require('node-fetch')
+const {
+  compose,
+  zipObj,
+  tap,
+  mapAwait,
+  tapLog,
+  prop,
+  gte,
+  map,
+  filter,
+  branch,
+  entries,
+} = require('./utils')
+
+// check for arg
+const date = process.argv?.[2]
+if (!date) {
+  throw new Error(
+    'Please call this script with a date in the YYYY-MM-DD format.'
+  )
+}
+
+// data
+const repos = [
```

Comment:
Can read these from https://github.com/WordPress/openverse/blob/main/data/github.yml

Reply:
I think I'd prefer not to add a dependency to parse yaml just for these few strings. If this project grows to include many more JS-based scripts, then I would be more willing to make sure we read from global constants.